test/activation_based/test_functional.py

```
@@ -0,0 +1,339 @@
+        reset_net(net)
+        assert net[1].v == 0.0
+    invalidate_reset_cache(net)
+
+
+def test_reset_net_preserves_parameters():
+    net = nn.Sequential(nn.Linear(4, 8), neuron.IFNode())
+    params_before = [p.clone() for p in net.parameters()]
+    reset_net(net)
+    try:
+        params_after = list(net.parameters())
+        for pb, pa in zip(params_before, params_after):
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)